config/prov/prov_test.yaml

```diff
@@ -186,6 +186,35 @@ k8s_cortx_deploy:
     deploy: 7200
     destroy: 900
     status: 120
+  thirdparty_resource:
+    server:
```

Comment:
Can you add the consul here(can be done as comment)

Reply:
done